packages/client/src/service/txpool.ts

```diff
@@ -177,6 +182,8 @@ export class TxPool {
 
     this.opened = false
     this.running = false
+
+    this.txGauge = options.txGauge
```

Comment:
Could we add a gauge for each transaction type instead of one general one?  That would be nice for seeing how many blob transactions are currently in the mempool (versus other types)

Reply:
I've added gauges for each different transaction type. Fixed 🙂.